zebrad/src/application.rs

```diff
@@ -135,10 +136,28 @@ impl Application for ZebradApp {
 
 impl ZebradApp {
     fn level(&self, command: &EntryPoint<ZebradCmd>) -> String {
-        if let Ok(level) = std::env::var("ZEBRAD_LOG") {
-            level
-        } else if command.verbose {
+        use ZebradCmd::*;
+        // Tracing sends output to stdout, so we disable it for some commands.
+        //
+        // TODO: send tracing output to stderr. This change requires an abscissa
+        //       update, because `abscissa_core::component::Tracing` uses
+        //       `tracing_subscriber::fmt::Formatter`, which has `Stdout` as a
```

Comment:
ouch, that sucks :(

Reply:
Yeah, it's also weird that the `tracing_subscriber::fmt::Formatter` default is stdout. I'd expect a logging library to use stderr by default, or ask the user to choose.